utils/k8s/k8s.go

```diff
@@ -660,3 +660,22 @@ func (k8s K8S) BlockUntilPodIsUpOrTimeout(pod *core_v1.Pod, timeout time.Duratio
 
 	return k8s.BlockUntilPodIsUpWithContext(ctx, pod)
 }
+
+// CreatePersistentVolumeClaim is use to create the PVC in the given name space
+func (k8s K8S) CreatePersistentVolumeClaim(persistentVolumeClaim *core_v1.PersistentVolumeClaim, ns string) (result *core_v1.PersistentVolumeClaim, err error) {
+	persistentVolumeClaimNameSpace := k8s.Clientset.CoreV1().PersistentVolumeClaims(ns)
+	result, err = persistentVolumeClaimNameSpace.Create(persistentVolumeClaim)
+	return
+}
+
```

Comment:
We can add one more `ListPersistentVolumeClaim` function to get a single one. Can you please add it?

Reply:
@sonasingh46 I think it is `GetPersistentVolumeClaim`.

Reply:
Yes... I missed there.